lib/signaling/v2/peerconnection.js

```diff
@@ -590,9 +600,15 @@ class PeerConnectionV2 extends StateMachine {
       this._queuedDescription = null;
       return this._maybeReoffer(this._peerConnection.localDescription);
     }).catch(error => {
-      throw error instanceof MediaClientRemoteDescFailedError
-        ? error
-        : new MediaClientLocalDescFailedError();
+      const errorToThrow = error instanceof MediaClientRemoteDescFailedError ? error : new MediaClientLocalDescFailedError();
+      if (errorToThrow instanceof MediaClientLocalDescFailedError) {
```

Comment:
@makarandp0 Why are we publishing only local sdp error?

Reply:
because the remote sdp error was handled on line 548, but I guess its better to handle them here, I will update.